docs/tutorials/elasticdl_local.md

```diff
@@ -12,37 +12,43 @@ In this tutorial, we use [hyperkit](https://github.com/moby/hyperkit) as the hyp
 
 We use TensorFlow Keras API to build our models. Please refer to this [tutorials](model_building.md) on model building for details.
 
+In this tutorial, we use a [model](https://github.com/sql-machine-learning/elasticdl/blob/develop/model_zoo/mnist_functional_api/mnist_functional_api.py) predefined in model zoo directory.
+
 ## Summit Job to Minikube
 
 ### Install ElasticDL
 
 ```bash
 git clone https://github.com/sql-machine-learning/elasticdl.git
 cd elasticdl
+pip install -r elasticdl/requirements.txt
 python setup.py install
 ```
 
-### Setup kubernetes related environment
+### Setup Kubernetes related environment
 
 ```bash
-minikube start --vm-driver=hyperkit --cpus 2 --memory 6144
+minikube start --vm-driver=hyperkit --cpus 2 --memory 6144 --disk-size=50gb
 kubectl apply -f elasticdl/manifests/examples/elasticdl-rbac.yaml
 eval $(minikube docker-env)
+export DOCKER_BUILDKIT=1
 bash elasticdl/docker/build_all.sh
 ```
 
 ### Summit a training job
 
-There are other docker settings that you might also want to configure prior to submitting the training job. Please run `minikube docker-env` to get docker host url and docker cert path.
+There are other docker settings to configure prior to submitting the training job. Please run `minikube docker-env` to get docker host url and docker cert path.
```

Comment:
There are other docker settings to configure prior to submitting the training job.

For example:

```bash
export DOCKER_BASE_URL=tcp://192.168.64.5:2376
export DOCKER_TLSCERT=${HOME}/.minikube/certs/cert.pem
export DOCKER_TLSKEY=${HOME}/.minikube/certs/key.pem
```
We can get these setting values by running `minikube docker-env`.

Reply:
Done